include/os/freebsd/spl/sys/time.h

```diff
@@ -62,6 +62,23 @@ typedef longlong_t	hrtime_t;
 #define	SEC_TO_TICK(sec)	((sec) * hz)
 #define	NSEC_TO_TICK(nsec)	((nsec) / (NANOSEC / hz))
 
+static __inline hrtime_t
+getlrtime(void)
+{
+	struct timespec ts;
+	hrtime_t nsec;
+
+	getnanotime(&ts);
+	nsec = ((hrtime_t)ts.tv_sec * NANOSEC) + ts.tv_nsec;
+	return (nsec);
+}
+
+static __inline void
+nanouptime(struct timespec *ts)
+{
+	clock(&ts);
+}
```

Comment:
FreeBSD already has `nanouptime()`. What's this?

Reply:
I... honestly don't know. I have no memory of writing that. Obivously it's wrong, I'll remove it.